bitnami/keycloak/values.yaml

```diff
@@ -1372,14 +1372,14 @@ externalDatabase:
 ## ref: https://www.keycloak.org/server/caching
 ## @param cache.enabled Switch to enable or disable the keycloak distributed cache for kubernetes.
 ## NOTE: Set to false to use 'local' cache (only supported when replicaCount=1).
-## @param cache.stackName Set infinispan cache stack to use
-## @param cache.stackFile Set infinispan cache stack filename to use
+## @param cache.stack Set infinispan cache stack to use, sets KC_CACHE_STACK (<https://www.keycloak.org/server/all-config?q=cache-stack>)
+## @param cache.configFile Set infinispan cache stack config filename sets KC_CACHE_CONFIG_FILE (<https://www.keycloak.org/server/all-config?q=cache-config-file>)
 ## @param cache.useHeadlessServiceWithAppVersion Set to true to create the headless service used for ispn containing the app version
 ##
 cache:
   enabled: true
-  stackName: kubernetes
-  stackFile: ""
+  stack: kubernetes
+  configFile: "cache-ispn.xml"
```

Comment:
Note that I explicitly set the configFile here as of https://www.keycloak.org/docs/latest/release_notes/index.html#infinispan-default-xml-configuration-location

With Keycloak versions >= 26.1.0 this should not be necessary anymore, but is maybe still better to be explicitly set then implicitly assumed.